server/src/test/resources/csars/yaml/valid/express/template.yml

```diff
@@ -31,6 +31,10 @@ topology_template:
           start:
             implementation:
               primary: simpletaskapp_js/start_stajs.sh
+            inputs:
+                REDIS_HOST: { get_attribute: [ redis_compute, private_address ] }
```

Comment:
Not sure if snakeyaml would complain, but these entries have wrong indentation

Reply:
Intellij was a bit stupid there